### ISSUE TITLE
[Feature request] Embed notebook in document

### ISSUE THREAD

**Issue by Ozzuke** [MaelImhof/obsidian-jupyter#100]
Would be really useful to be able to embed a notebook into an obsidian document in a similar way to how a PDF file can be embedded.

**Comment by MaelImhof** [comment]
Do you mean using something like `![[Jupyter notebook filename.ipynb]]`, similarly to how you can [embed notes and files inside of other notes](https://help.obsidian.md/Linking+notes+and+files/Embed+files#Embed+a+PDF+in+a+note) (see below)?

![image](https://github.com/user-attachments/assets/d0111c73-c1f5-4f84-b874-6d24bca9ada2)

**Comment by Ozzuke** [comment]
Yes exactly that, though I think it would look similar to how a PDF file is embedded (image added below).

<img width="663" alt="Screenshot 2024-09-19 at 21 32 42" src="https://github.com/user-attachments/assets/38bde83a-7b0a-435a-9c70-1f12c935e69e">

**Comment by MaelImhof** [comment]
Right I see, that would indeed be great, unfortunately I do not have much time to work the plugin at the moment, I'll start again in January. Hopefully this feature is as simple as adding a webview or something similar.

Thanks for the suggestion !

**Comment by IDZorin** [comment]
that would be killer feature indeed
